package-lock.json

```diff
@@ -1092,8 +1092,7 @@
             "bindings": {
               "version": "1.3.0",
               "resolved": "https://registry.npmjs.org/bindings/-/bindings-1.3.0.tgz",
-              "integrity": "sha512-DpLh5EzMR2kzvX1KIlVC0VkC3iZtHKTgdtZ0a3pglBZdaQFjt5S9g9xd1lE+YvXyfd6mtCeRnrUfOLYiTMlNSw==",
-              "dev": true
```

Comment:
Any Idea why this would have changed?

Reply:
cc @julien51

Reply:
This is something I have been trying to understand for weeks. npm is weird. this has no impact.

Reply:
Ok, thx.